BankManagment/BankManagment/Controllers/AccountTypeController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using BankManagmentDB.Entity;
+using BankManagmentDB;
+using Microsoft.AspNetCore.Mvc;
+
+
+namespace BankManagment.Controllers
+{
+    [Route("api/AccountType")]
+    [ApiController]
+    public class AccountTypeController : ControllerBase
+    {
+        private readonly IUnitOfWork _unitOfWork;
+
+        public AccountTypeController(IUnitOfWork unitOfWork)
+        {
+            _unitOfWork = unitOfWork;
```

Comment:
Unit of Work object should not be accessible inside the ccontroller method, instead create service and use unit of work inside that service